### ISSUE TITLE
"Send to Open Radar" toggle unselectable when tapping out of the window

### ISSUE THREAD

**Issue by ChrisixFlash** [br1sk/brisk#131]
**Steps to reproduce:**
1. Open a new radar window in brisk ("Send to Open Radar" toggle is selected)
2. Tap out of the window (to copy some text etc)
3. Tap into brisk again

The "Send to Open Radar" toggle is now greyed out and can't be selected again. Also happens when adding an attachment.

![screen shot 2018-01-30 at 11 13 45 am](https://user-images.githubusercontent.com/21023299/35560952-348620ae-05af-11e8-9ce2-c73b72746596.png)

**Comment by keith** [comment]
Do you have a token setup in the preferences?

**Comment by ChrisixFlash** [comment]
@keith Nope, nothing set up. Just installed it as told in the readme and then opened it from the applications folder.

**Comment by ChrisixFlash** [comment]
Keith, I found the issue. The API key in the preferences didn't seem to match for some reason. It's now working as intended and doesn't grey out anymore.

So the real bug would be: "Send to Open Radar" toggle is available without a correct API Key set in the preferences.

**Comment by whoinoi** [comment]
I second this issue. I used the app for the first time and didn't know that you'd need to register an account at Open Radar. The active toggle made it look like you don't need to. Then after app switching the toggle becomes inactive which I didn't understand why till I read this issue.

Also when first opening the app the "Show dock icon" setting is disabled, which makes it impossible to access settings other than with the keyboard command CMD + , which makes it even more difficult to know that you need an API Key from Open Radar for the app to submit to Open Radar.

I guess the Readme should include some setup details.
